drivers/tee/optee/optee_smc.h

```diff
@@ -215,6 +215,9 @@ struct optee_smc_get_shm_config_result {
  */
 #define OPTEE_SMC_SEC_CAP_DYNAMIC_SHM		BIT(2)
 
+/* Secure world supports Shared Memory with a NULL Reference */
+#define OPTEE_SMC_SEC_CAP_MEMREF_NULL		BIT(3)
```

Comment:
We have `OPTEE_SMC_SEC_CAP_VIRTUALIZATION` on that bit, so this must be change to BIT(4).

Reply:
I guess we should define `OPTEE_SMC_SEC_CAP_VIRTUALIZATION` in this file.

s/Reference/buffer reference/

Reply:
Fixed OPTEE_SMC_SEC_CAP_MEMREF_NULL to BIT(4) and comment